gulpfile.js

```diff
@@ -33,6 +33,10 @@ const NODE_MODULES_PATH = ('NODE_PATH' in process.env) ?
     process.env.NODE_PATH.split(':')[0] :
     './node_modules/';
 
+function shouldLint(opt, task) {
+  return require('./debug-config.json')[opt] ? [task] : [];
```

Comment:
Continuing on from the notion of a `debug-config.json-dist`, this code should get wrapped in a try/catch so that defaults are used if the `debug-config.json` file is missing.